src/common/i18n/translations/en.ts

```diff
@@ -1,6 +1,17 @@
 import { Translations } from "./type";
 
 export const en: Translations = {
+  redemptionStats: {
+    youDistributed: "You distributed",
+    lastDistributedAt: "Last distributed at ",
+    distributedTime: "%{distributionTime} ",
+
+    back: "Back",
```

Comment:
I personally don't see the value of separating the sentence. Let me know what you think.
I think we can just do this:

```suggestion
    lastDistributedAt: "Last distributed at %{lastTransactionTime}",
    back: "Back",
```